mods/species/vatborn/datum/species.dm

```diff
@@ -0,0 +1,49 @@
+/datum/appearance_descriptor/age/vatborn
+	name = "age"
+	chargen_max_index = 9
+
+	standalone_value_descriptors = list( //vatborns grow quickly and die young
+		"an infant" =      1,
+		"a toddler" =      2,
+		"a child" =        3,
+		"a teenager" =    4,
+		"a young adult" = 6,
+		"an adult" =      20,
+		"middle-aged" =   35,
+		"aging" =         50,
+		"elderly" =       70,
+		"ancient" =      91
+	)
+
+
+/decl/species/human/vatborn
+	name = SPECIES_VATBORN
+	name_plural = SPECIES_VATBORN
```

Comment:
```suggestion
/decl/species/human/vatborn
	uid = "species_vatborn"
	name = "Vatborn"
	name_plural = "Vatborns"
```

This was just refactored, very minor impact for vatborn since they aren't whitelisted or anything!